composer.json

```diff
@@ -21,12 +21,12 @@
     "extra": {
     },
     "require": {
-        "php": "^7.3 || ~8.0.0",
+        "php": "^7.3 || ~8.0.0 || ~8.1.0",
         "ext-ldap": "*",
         "laminas/laminas-zendframework-bridge": "^1.1"
```

Comment:
This dependency should be dropped too: it's a relatively simple change, please see https://github.com/laminas/technical-steering-committee/blob/c7f6babfb1773d9cb9371208102cbb8b27980bd5/meetings/minutes/2021-08-02-TSC-Minutes.md#remove-laminaslaminas-zendframework-bridge-dependency-from-our-packages

Reply:
It is still in the lockfile because of:
```
php composer.phar why laminas/laminas-zendframework-bridge
laminas/laminas-config        3.5.0  requires  laminas/laminas-zendframework-bridge (^1.0)  
laminas/laminas-eventmanager  3.3.1  requires  laminas/laminas-zendframework-bridge (^1.0)  
laminas/laminas-stdlib        3.3.1  requires  laminas/laminas-zendframework-bridge (^1.0)  
```

Should I bump the versions of those packages to completely remove the dependency?

Reply:
No, it's fine - at least we got rid of the dependency from here :-) 

Further upgrades will slowly phase it out everywhere :+1: